### PR TITLE
Update RELEASE.md

### PR DESCRIPTION
**Pull request by sunanda690**

### CODE REVIEW

File: docs/RELEASE.md [hunk 1]
```diff
@@ -13,5 +13,6 @@ vx.y.z
    * Go to 'Releases' tab
    * Create new draft
    * Select tag and binaries(if  needed)
+       1. generate linux installer: `pkg -t node17-linux-x64 dist/src/index.js -o secretsfoundry-{version}-linux-x64`
```
Comment (shubham-rai-tf) [inline docs/RELEASE.md:16]: This is to be generated locally? If so does this need to be uploaded somewhere?

Comment (sunanda690) [reply docs/RELEASE.md:16]: yeah you get to upload files when drafting the release

Comment (chiragjn) [reply docs/RELEASE.md:16]: we shouldn't need this unless we want to maintain distribution on Github outside of npm

Comment (shubham-rai-tf) [reply docs/RELEASE.md:16]: We do need a binary which can be directly called. npm is not present inside argocd where this is called

**Review by shubham-rai-tf — APPROVED** [review]

**Pull request merged**